### ISSUE TITLE
Add scaler transformers to Pyreal

### ISSUE THREAD

**Issue by zyteka** [sibyl-dev/pyreal#432]
Pyreal currently supports scaling data transforms through the `DataFrameWrapper` class, which allows us to use sklearn scalers. However, because scaling data is such a common process, pyreal code would look a lot cleaner if it directly supported these processes. 

For this PR, we will add three new Transformer classes to pyreal, in a separate transformer module called `scale`. These most likely can be implemented by just wrapping `DataFrameWrapper` instances. They should take in the same parameters as the sklearn versions.
1. `MinMaxScaler` - wraps [sklearn.preprocessing.MinMaxScaler](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html#sklearn.preprocessing.MinMaxScaler)
    - Include optional `feature_range` and `clip` parameters that get passed to wrapped transformer
2. `Normalizer` - wraps [sklearn.preprocessing.Normalizer](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Normalizer.html#sklearn.preprocessing.Normalizer)
    - Include optional `norm` parameter that gets passed to wrapped transformer
3. `StandardScaler` - wraps [sklearn.preprocessing.StandardScaler](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html#sklearn.preprocessing.StandardScaler)
    - Include the `with_mean` and `with_std` parameters that get passed to wrapped transformer
